### ISSUE TITLE
irregular shutdown on dual monitor setup - freezing

### ISSUE THREAD

**Issue by rogierfvv** [linuxmint/Cinnamon#7780]
Ever since kernel version 4.13, there have been irregular shutdowns. Not always, but regularly - basically after the screen goes blank, I cannot restart and it seems the desktop becomes consolidated on one monitor before it falls asleep....

in 4.15.0-24 it was very bad and I went back to a prior version. Presently, it's a little better. It still happens reguarly, but not all the time any longer.

```
 * Cinnamon version (cinnamon --version) 3.6.6
 * Distribution - (Mint 17.2, Arch, Fedora 25, etc...) Mint 18.3
 * Graphics hardware *and* driver used  AMD/ATI device 676df
 * 64 bit 
 ```

**Issue**

**Steps to reproduce**

**Expected behaviour**

**Other information**

**Comment by icarter09** [comment]
@rogierfvv are you still experiencing this issue?

**Comment by rogierfvv** [comment]
Indeed, it continues. Now under Mind 19.2. In the shutdown process, somehow, all the browsers are moved to the primary screen, so that, when I reactivate the machine, first I have to rearrange the two browsers I normally have on the secondary screen. There appears to be no way to preserve the layout when the machine falls asleep.

**Comment by icarter09** [comment]
@rogierfvv thanks for the update.
